pkgs/applications/misc/gcfflasher/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, pkgs
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcfflasher";
+  version = "4.0.3-beta";
+
+  src = fetchFromGitHub {
+    owner = "dresden-elektronik";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-m+iDBfsHo+PLYd3K8JaKwhIXcnj+Q8w7gIgmHp+0plk=";
+  };
+
+  nativeBuildInputs = with pkgs; [ gcc pkg-config libgpiod ];
```

```suggestion
  nativeBuildInputs = [
    pkg-config
  ];
  
  buildInputs = [
    libgpiod
  ];
```

Here are my thoughts about this suggestion:
- the compiler is always part of the stdenv you are consuming
- libraries are likely needed at runtime, so they should be in `buildInputs` instead
- don't import dependencies from `pkgs`, mention them as explicit function arguments
- expand lists to one item per line, so future changes will less noisy